src/charm_state.py

```diff
@@ -119,10 +138,12 @@ def from_charm(cls, charm: CharmBase) -> "CharmState":
         workers = charm.config.get("webserver_workers")
         threads = charm.config.get("webserver_threads")
         flask_config = {
-            k.removeprefix("flask_"): v for k, v in charm.config.items() if k.startswith("flask_")
+            k.removeprefix("flask_"): v
```

Comment:
I'd document that behavior in the docstring on dealing with variables.

Reply:
Added, thanks!